content/F2015/F15-event-career-panel.md

```diff
@@ -0,0 +1,21 @@
+title: CSC and WiCS Career Panel
+Date: 2015-09-24 17:00-19:00
+Category: Events
+Tags: panels, industry
+Slug: career-panel-F15
+Author: Evy Kassirer
+Summary: A panel of alumni working in the industry share their experiences and advice.
+
+CSC is joining WiCS to host a career panel! Come hear from Waterloo alumni as 
+they speak about their time at Waterloo, experience with coop, and life beyond 
+university. A great chance to network and seek advice!
+
+Snacks will be provided.
+
+## Event Details ##
+
++ **Who:** Everyone!
```

Comment:
Yes, that would be good - I was hoping to do that :)
(And hopefully we know who panelists are soon!)